widget/templates/_3Dmanifest.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "<%= widgetName %>",
+  "2D": <%= is2d %>,
+  "3D": <%= is3d %>,
+  "platform": "HTML3D",
+  "version": "2.0beta",
+  "wabVersion": "2.0beta",
+  "author": "<%= author %>",
+  "description": "<%= description %>",
+  "copyright": "",
+  "license": "<%= license %>",
+  "properties": {
+    "inPanel": <%= inPanel %>,
+    "hasConfig": <%= hasConfig %>,
+    "hasUIFile": <%= hasUIFile %>,
```

Comment:
I wonder if the other properties are omitted here just b/c they default to true (since WAB v1.2). I would assume they all still apply to 3d widgets.